rosidl_generator_cpp/CMakeLists.txt

```diff
@@ -11,12 +11,56 @@ ament_export_include_directories(include)
 ament_python_install_package(${PROJECT_NAME})
 
 if(AMENT_ENABLE_TESTING)
-  find_package(ament_lint_auto REQUIRED)
-  ament_lint_auto_find_test_dependencies()
```

Comment:
OK, I will remove the other `find_packages` too since `ament_lint_auto_find_test_dependencies()` will find them for me, correct?